sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -1420,18 +1420,20 @@ class DatasetSuite extends QueryTest
     dataset.sparkSession.catalog.dropTempView("tempView")
 
     withDatabase("test_db") {
```

Comment:
We're cleaning the temp db up so we're good?

Reply:
> We're cleaning the temp db up so we're good?

`tempView` does not seem to be cleaned up because the temporary view does not belong to a db.